app/importers/concerns/taxonomy_methods.rb

```diff
@@ -0,0 +1,35 @@
+module TaxonomyMethods
+  attr_accessor :taxonomy_parser
+
+  def get_missing_country(mappable_field)
+    country_array = normalize_industry('Missing Country: ' + mappable_field)
+    country_array.nil? ? nil : lookup_country(country_array.first)
+  end
+
+  def set_taxonomy_parser
+    self.taxonomy_parser = TaxonomyParser.new(Rails.configuration.frozen_protege_source)
+    taxonomy_parser.concepts = YAML.load_file(Rails.configuration.frozen_taxonomy_concepts)
+  end
```

Comment:
Rather than dusting the importers with `set_taxonomy_parser` calls, better to lazily instantiate. Something like:

```
def taxonomy_parser
  unless self.taxonomy_parser
    self.taxonomy_parser = TaxonomyParser.new(Rails.configuration.frozen_protege_source)
    taxonomy_parser.concepts = YAML.load_file(Rails.configuration.frozen_taxonomy_concepts)
  end
  self.taxonomy_parser
end
```